PhoneBook.Doc415/EmailSender.cs

```diff
@@ -0,0 +1,58 @@
+﻿using PhoneBook.Doc415.Models;
+using Spectre.Console;
+using System.Net;
+using System.Net.Mail;
+namespace PhoneBook.Doc415;
+
+internal class EmailSender
+{
+    string smtpServer;
+    string userName;
+    string password;
+    string subject;
+    string body;
+    string recipentEmail;
+    string myEmail;
+    public void SetUpSender()
```

Comment:
why EmailSender responsibility it's taking information from within (here its user). Data that EmailSender requires to send messages should be passed upon creation. 

This class breaks Single Responsibility Principle

Reply:
Didnt spend much time on it. Just rushed to finish. I ll make major revision on challenges part.  One class one job...